assets/css/styles.css

```diff
@@ -54,3 +54,8 @@ h6,
 .outline-menu > .outline-list {
     padding-left: 0;
 }
+@media only screen and (min-width: 576px) {
```

Comment:
added this so that the coffee chat card on the services screen only takes up half the row instead of the entire row.